README.md

```diff
@@ -430,7 +430,7 @@ barion.refundPayment({
 });
 ```
 
-### Send money to bank account - barion.bankTransfer(options, \[callback\])
+### Send money to a bank account - barion.bankTransfer(options, \[callback\])
```

Comment:
Can you fix it on the Table of Contents too?

Reply:
of course!